Sloth.Core/Extensions/StringExtensions.cs

```diff
@@ -19,5 +21,39 @@ public static string Base64Decode(this string base64EncodedData)
             var base64EncodedBytes = System.Convert.FromBase64String(base64EncodedData);
             return System.Text.Encoding.UTF8.GetString(base64EncodedBytes);
         }
+
+        public static string SafeTruncate(this string value, int max)
+        {
+            if (string.IsNullOrWhiteSpace(value) || value.Length <= max)
+            {
+                return value;
+            }
+
+            if (max <= 0)
+            {
+                throw new ArgumentOutOfRangeException();
+            }
+
+            return value.Substring(0, max);
+        }
+
+        public static string SafeRegexRemove(this string value, string regEx = @"[^0-9a-zA-Z\.\-\' ]+")
```

Comment:
this is neat

Reply:
But it didn't work like I expected... See SafeRegexRemoveNot()